flixel/FlxObject.hx

```diff
@@ -612,7 +612,12 @@ class FlxObject extends FlxBasic
 	 * Set this to `false` if you want to skip the automatic motion/movement stuff (see `updateMotion()`).
 	 * `FlxObject` and `FlxSprite` default to `true`. `FlxText`, `FlxTileblock` and `FlxTilemap` default to `false`.
 	 */
-	public var moves(default, set):Bool = true;
+	public var moves(default, set):Bool = movesDefault;
+
+	/**
+	 * The default value of the `moves` variable.
+	 */
+	public static var movesDefault:Bool = true;
```

Comment:
Couple notes:
- static vars should come before instance vars. this should be moved so it is just after [defaultPixelPerfectPositioning](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/FlxObject.hx#L80) which is on line 80
- the convention for these types of vars is `defaultMoves` not `movesDefault`
- lets change the comment to ```The default `moves` value of all future `FlxObjects` ```
    - no period on single sentence comments
    - avoid the word variable (i know `defaultPixelPerfectPosition` says "var" thats also bad
- I'm trying to move a lot of existing static vars to FlxG. I find these to be esoteric, hard to find and a little confusing. That said, there's little harm in adding one more and this could actually retroactively improve performance dramatically in many games